main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_name:
+        data = json.load(file_name)
+
+    for name, player in data.items():
+        info_race = player["race"]
+        if not Race.objects.filter(name=info_race["name"]).exists():
+            Race.objects.create(
```

Comment:
Use get_or_create 
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create

Reply:
Fix in all places